custom_components/switchbot-curtain/const.py

```diff
@@ -0,0 +1,16 @@
+"""Constants for the switchbot integration."""
+
+DOMAIN = "switchbot-curtain"
+MANUFACTURER = "switchbot"
+
+ATTR_CURTAIN = "curtain"
+ATTR_BOT = "bot"
+DEFAULT_NAME = "Switchbot"
+DEFAULT_RETRY_COUNT = 3
+DEFAULT_RETRY_TIMEOUT = 1
+DEFAULT_TIME_BETWEEN_UPDATE_COMMAND = 10
```

Comment:
@RenierM26 isn't the DEFAULT_TIME_BETWEEN_UPDATE_COMMAND to fast = 10 seconds. I think that we can keep default to 5 minutes ?

Reply:
or 10 = 10 minutes ... ?
<img width="385" alt="Screenshot 2021-05-27 at 18 42 15" src="https://user-images.githubusercontent.com/1692861/119860642-36403100-bf1f-11eb-9053-59ab99ef47a6.png">

Reply:
After looking into pySwitchbot code, time between updates = waiting time to get newer information after a command was issued to the bot. (I think that we need to wait let's say 10 seconds after bot moved to correct position and a scanned is performed)

https://github.com/Danielhiversen/pySwitchbot/blob/1357c7c54403296172ad51fa3db2b3748b0e56f5/switchbot/__init__.py#L124

In order to control polling time, we need to update SCAN_INTERVAL which is now to 35 seconds ...

So I think that "time between updates" is misleading in this context ...

Reply:
Hi @alextud,

I've also noticed that. I'm busy looking into a wat to set SCAN_INTERVAL the same as DEFAULT_TIME_BETWEEN_UPDATE_COMMAND. The hassio update and async_update functions have a hard coded warning if the update takes longer than 10 seconds. (in hassio entity helpers code)

I'm thinking of changing polling to False and scheduling ha updates based on DEFAULT_TIME_BETWEEN_UPDATE_COMMAND.

Reply:
https://github.com/home-assistant/core/blob/3ca7eb9440f8b6b4b4825b8b550903a699b3de03/homeassistant/helpers/entity.py#L527

Reply:
You are correct about "time between updates".  Will need to factor this in as well.